### PR TITLE
[NFC] Add metadata to describe personal note field

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Add metadata to pcp form

Before
----------------------------------------
no html metadata

After
----------------------------------------
html metadata

Technical Details
----------------------------------------
Just follows our standards of adding metadata

Comments
----------------------------------------
This is towards #12044 - however an additional change similar to below would be needed to make it work - I've stopped short of adding that because it would require additional issue replication which I haven't done

```
diff --git a/CRM/PCP/BAO/PCP.php b/CRM/PCP/BAO/PCP.php
index e8d3dac..b233e4b 100644
--- a/CRM/PCP/BAO/PCP.php
+++ b/CRM/PCP/BAO/PCP.php
   public static function buildPcp($pcpId, &$page, &$elements = NULL) {
@@ -453,7 +453,7 @@ WHERE pcp.id = %1 AND cc.contribution_status_id =1 AND cc.is_test = 0";
       $page->_defaults['pcp_is_anonymous'] = 0;
 
       $page->add('text', 'pcp_roll_nickname', ts('Name'), array('maxlength' => 30));
-      $page->add('textarea', "pcp_personal_note", ts('Personal Note'), array('style' => 'height: 3em; width: 40em;'));
+      $page->addField("pcp_personal_note", ['context' => 'create', 'entity' => 'Pcp', 'name' => '"pcp_personal_note'], array('style' => 'height: 3em; width: 40em;'));
     }
     else {
       $page->assign('is_honor_roll', FALSE);


```

### CODE REVIEW

File: xml/schema/Contribute/ContributionSoft.xml [hunk 1]
```diff
@@ -129,6 +129,9 @@
     <type>varchar</type>
     <title>Soft Contribution PCP Note</title>
     <length>255</length>
+    <html>
+      <type>Text</type>
```
Comment (seamuslee001) [inline xml/schema/Contribute/ContributionSoft.xml]: @eileenmcnaughton should the type not be TextArea?

Comment (eileenmcnaughton) [reply xml/schema/Contribute/ContributionSoft.xml]: hmm - maybe - why was it text on the other note field - yes will change

Comment (eileenmcnaughton) [reply xml/schema/Contribute/ContributionSoft.xml]: done

Comment (seamuslee001) [reply xml/schema/Contribute/ContributionSoft.xml]: I think that was a text field where as this is a rich text field perhaps?

**Comment by eileenmcnaughton** [comment]
merging per Seamus merge-on-pass

**Pull request merged**